src/extension/common.ts

```diff
@@ -3,9 +3,15 @@ import { Unport } from 'unport'
 import { workspace } from 'vscode'
 import type { ChildProcessWithoutNullStreams } from 'node:child_process'
 import { execFile } from 'node:child_process'
+import { compare } from 'compare-versions'
```

Comment:
I don't think we need another library for feature detection. When having errors, just prompting users to download the latest version is fine.